### ISSUE TITLE
Inputs in imageToImage are not working

### ISSUE THREAD

**Issue by mmxhello** [huggingface/huggingface.js#262]
There is an example in the docs: 
![image](https://github.com/huggingface/huggingface.js/assets/60787898/7123e890-c0aa-490c-b594-b7c33775787d)

Here is my code: 
![image](https://github.com/huggingface/huggingface.js/assets/60787898/530f7424-4ad8-4871-baa6-fcb6b9b0af02)

It does not work: 
![image](https://github.com/huggingface/huggingface.js/assets/60787898/1911a740-88fe-4ce7-ac2d-1e8797ece05e)

Even like this there is the same error: 
![image](https://github.com/huggingface/huggingface.js/assets/60787898/e95a4199-9e08-4d65-89e3-f53f7d18a38c)
or 
![image](https://github.com/huggingface/huggingface.js/assets/60787898/0ebfecc7-e45a-4631-9bb3-22cac07d1a29)

This way from the hugging face docs also does not seem to work: 
![image](https://github.com/huggingface/huggingface.js/assets/60787898/7244d38e-d5d2-4e54-a81c-486910ed59ec)

**Comment by mmxhello** [comment]
Thanks! That works. 

There were also some issues: 

'prompt' should be passed directly to the function, but not inside 'parameters', or there will be an error: 
 __call__() got multiple values for argument 'prompt'

And i added 'utf-8' to the readFileSync like this: 
inputs: new Blob([fs.readFileSync(photoFile, 'utf-8')])
Otherwise it throwed an error ''utf-8' codec can't decode byte 0xff in position 0: invalid start byte'

The final code looks like this: 
![image](https://github.com/huggingface/huggingface.js/assets/60787898/0944707c-a81d-4db7-8e1d-5395aad554bc)

Thank you for your answer and i think it would be good to update the docs

**Comment by mmxhello** [comment]
Hm, strange, it is no longer throwing this error. The image was 
![test3](https://github.com/huggingface/huggingface.js/assets/60787898/5614d0d0-8cbf-4c87-943d-e63799e7ecbe)

**Comment by mmxhello** [comment]
There is an another strange error, btw: 
'float' object cannot be interpreted as an integer

![image](https://github.com/huggingface/huggingface.js/assets/60787898/5808ad7a-299a-4d42-a25b-bfd0ec2bc65e)

It accepts only integers in parameters

**Comment by mmxhello** [comment]
You are right, the model used in your tutorial works well with both options - Blob and readFileSync. Thank you for noticing. I'll try to find docs for the models i need

**Comment by radames** [comment]
hi @mmxhello , our API inference only has one task per model, since many SD models allow multiple tasks per model you can't run image to image on a text to image defined model. One solution is to duplicate the model and switch the task tag, here is one model SD2-1 that works with image to image , fell free to test and use with huggingface.js, switch model: `stabilityai/stable-diffusion-2-1` to `radames/stable-diffusion-2-1-img2img`
https://huggingface.co/radames/stable-diffusion-2-1-img2img
